dftimewolf/lib/exporters/gce_image_from_disk.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+"""Create disks in GCE from disk images."""
+
+from typing import Any, Optional, Type, Union
+
+from libcloudforensics.providers.gcp.internal import project as gcp_project
+from libcloudforensics.providers.gcp.internal.compute import GoogleComputeDisk
+from dftimewolf.lib import module
+from dftimewolf.lib.containers import containers, interface
+from dftimewolf.lib.modules import manager as modules_manager
+from dftimewolf.lib.state import DFTimewolfState
+
+
+class GCEImageFromDisk(module.ThreadAwareModule):
+  """Create images in GCE from disks."""
+
+  def __init__(self,
+               state: DFTimewolfState,
+               name: Optional[str]=None,
+               critical: bool=False) -> None:
+    """Initialises creating disks in GCE from disk images.
+
+    Args:
+      state (DFTimewolfState): recipe state.
+      name (Optional[str]): The module's runtime name.
+      critical (Optional[bool]): True if the module is critical, which causes
+          the entire recipe to fail if the module encounters an error.
+    """
+    super(GCEImageFromDisk, self).__init__(
+        state, name=name, critical=critical)
+    self.source_project: str = ''
+    self.source_zone: str = ''
+    self.dest_project: str = ''
+    self.dest_zone: str = ''
+
+  # pylint: disable=arguments-differ
+  def SetUp(self,
+      source_disks: Union[str, None] = '',
+      source_project: str = '',
+      source_zone: str = '',
+      destination_project: str = '',
+      destination_zone: str = '') -> None:
```

Comment:
Actually, the destination isn't required, (if not specified, dest values are copied from source) but removed the defaults for the remaining.

Reply:
Yeah, but recipes will still have to provide all arguments to `SetUp` (otherwise the test checking for this will fail). This was done to reduce the places where defaults are specified (they are now, or should be, always specified in the recipes rather than the modules themselves)

Reply:
Ahh yes, I see what you mean. Updated.